INSTALLATION.md

```diff
@@ -43,6 +43,12 @@ You'll need to set up the IDE and mobile device emulator on your local system an
 
 # Installation
 
+You can start using Talawa by any of the two methods:
+
+1. Install the `apk` provided in the release section of this repo. It is built against the latest codebase. Please note that the release is provided only for Android. For iOS, you will still need to build the app yourself.
+2. Manually build the app by below described steps.
+   
+
```

Comment:
This doesn't flow. There needs to be heading below stating that this is the manual approach.
There needs to be another section which explains how to use the APK approach.

What we have is insufficient

Reply:
1. Okay I'll add a heading.
2. Use of an APK is a very common thing. The user just needs to click on the file and then click on install. Even a normal user knows this these days. Even if I try to write something, it will be like "Click on the file, and choose to download when prompted for. That will download the apk. Then click on the downloaded apk and then click on install. Once installed, go to your apps section and you will find Talawa there".
My point being that newcomers will already know all of this, and we don't need to clutter our `.md` files. 
Please think about this and let me know if you still think that a detailed description is required. I will add it.

Reply:
Moreover @palisadoes I have planned for changes to the release workflow like "automatic stable weekly release" etc. We can tackle the documentation part then too.

Reply:
> 1. Okay I'll add a heading for the manual build.
> 
>     2. Use of an APK is a very common thing. The user just needs to click on the file and then click on install. Even a normal user knows this these days. Even if I try to write something, it will be like "Click on the file, and choose to download when prompted for. That will download the apk. Then click on the downloaded apk and then click on install. Once installed, go to your apps section and you will find Talawa there".
>        My point being that newcomers will already know all of this, and we don't need to clutter our `.md` files.
>        Please think about this and let me know if you still think that a detailed description is required. I will add it.

You don't know for sure. Our Outreachy experience overturned all our prior assumptions about end users. Even this simple wording will be helpful.

Reply:
Think of real Talawa Admins using the MVP having to explain this to all their members from children to grand parents across all spectra of society.

Reply:
Our documentation has to be really good as we don't have a help desk as an open source project.

Reply:
Understood. Changing it now :)

Reply:
@palisadoes just added quite some information about `apk` installation. Please review and merge now :)
I will add more documentation if necessary in the subsequent PRs about stable releases.